README.md

```diff
@@ -22,27 +22,14 @@ Build the `sentinel-visor` binary to the root of the project directory:
 $ make build
 ```
 
-Install PostgreSQL:
+Install TimescaleDB v1.7.4:
 
-```sh
-brew install postgresql@12
-```
-
-Install TimescaleDB:
-
-```sh
-brew tap timescale/tap
-brew install timescaledb
-```
-
-You may need to run the following _before_ installing timescale to get it to compile:
```

Comment:
Note that homebrew no longer makes v1.7 available for download via this method. Changed to use docker-compose as recommended means for getting the correct (and tuned) versions of Postgres/TimescaleDB in the least amount of effort.